JetStreamDriver.js

```diff
@@ -1870,7 +1879,7 @@ const BENCHMARKS = [
         ],
         iterations: 160,
         worstCaseCount: 16,
-        testGroup: BigIntMiscGroup,
+        tags: ["BigIntMisc",]
```

Comment:
Nit: some of these have trailing commas some don't.

Reply:
let me go with trailing commas everywhere!